Yafc/Widgets/MainScreenTabBar.cs

```diff
@@ -109,29 +109,35 @@ private void BuildContents(ImGui gui) {
     private void PageRightClickDropdown(ImGui gui, ProjectPage page) {
         bool isSecondary = screen.secondaryPage == page;
         bool isActive = screen.activePage == page;
-        if (gui.BuildContextMenuButton(LSs.EditPageProperties)) {
+        if (gui.BuildContextMenuButton(LSs.EditPageProperties, icon:Icon.Edit)) {
```

Comment:
```suggestion
        if (gui.BuildContextMenuButton(LSs.EditPageProperties, icon: Icon.Edit)) {
```
Our code style calls for spaces here, and missing spaces will cause problems for those of us who are using the recommended [pre-push hook](https://github.com/shpaass/yafc-ce/blob/master/.githooks/pre-push).